Formula/imagemagick.rb

```diff
@@ -72,49 +72,149 @@ def install
       --enable-static
     ]
 
+    if build.without? "freetype"
```

Comment:
It feels a bit weird to reverse the order here to `if build.without?` rather than `if build.with?`, particularly since we switch half way through the formula here. I'd maybe prefer we kept them using the same format to avoid contributors trying to change stuff quickly not realising they change layout half way though.